packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +879,158 @@ describe('serverless cluster', () => {
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      securityGroups: [sg],
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
+      Engine: 'aurora-postgresql',
+      DBClusterParameterGroupName: 'default.aurora-postgresql10',
+      EngineMode: 'serverless',
+      MasterUsername: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:username::}}',
+          ],
+        ],
+      },
+      MasterUserPassword: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:password::}}',
+          ],
+        ],
+      },
+      VpcSecurityGroupIds: ['SecurityGroupId12345'],
+    });
+  });
+
+  test('can create a Serverless cluster without vpc but with imported subnet group', () => {
+    // GIVEN
+    const stack = testStack();
+    const subnetGroup = SubnetGroup.fromSubnetGroupName(stack, 'SubnetGroup12345', 'SubnetGroupId12345');
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+      subnetGroup,
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
+      Engine: 'aurora-postgresql',
+      DBClusterParameterGroupName: 'default.aurora-postgresql10',
+      EngineMode: 'serverless',
+      DBSubnetGroupName: { Ref: 'SubnetGroup12345' },
+      MasterUsername: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:username::}}',
+          ],
+        ],
+      },
+      MasterUserPassword: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:password::}}',
+          ],
+        ],
+      },
+      VpcSecurityGroupIds: ['SecurityGroupId12345'],
+    });
+  });
+
+  test('can\'t create a Serverless cluster without vpc but with imported vpc subnets', () => {
+    // GIVEN
+    const stack = testStack();
+
+    // WHEN
+    const vpcSubnets = {
+      subnetName: 'AVpcSubnet',
+    };
+
+    // THEN
+    expect(() => new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
```

Comment:
Again, let's use either Postgres with a version, or a different Engine that has a default major version instead of specifying `parameterGroup`.

Reply:
👍🏻